<?xml version="1.0" encoding="UTF-8"?>
<robot generated="20181127 19:08:08.995" generator="Robot 3.0.2 (Python 2.7.15 on win32)">
<suite source="C:\D\plc_system_test" id="s1" name="Plc System Test">
<suite source="C:\D\plc_system_test\tc.4.robot" id="s1-s1" name="Tc.4">
<test id="s1-s1-t1" name="tc_4.1">
<kw type="setup" name="Init Test" library="plc_tb_ctrl">
<msg timestamp="20181127 19:08:09.227" level="INFO">start detail logging</msg>
<status status="PASS" endtime="20181127 19:08:14.228" starttime="20181127 19:08:09.227"></status>
</kw>
<kw name="Run Iot Test" library="plc_tb_ctrl.PlcSystemTestbench">
<arguments>
<arg>tc_4_1</arg>
</arguments>
<msg timestamp="20181127 19:08:14.243" level="INFO">Open concentrator port</msg>
<msg timestamp="20181127 19:08:14.243" level="INFO">Close TB port</msg>
<msg timestamp="20181127 19:08:14.462" level="INFO">Open TB port</msg>
<msg timestamp="20181127 19:08:14.462" level="INFO">power off all of meters and cco, delay 5s</msg>
<msg timestamp="20181127 19:08:19.473" level="INFO">power on all of meters and cco. delay 5s for boot</msg>
<msg timestamp="20181127 19:08:24.489" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.489" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.489" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.504" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.504" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.504" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.504" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.520" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.520" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.520" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.520" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.520" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.520" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.520" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.536" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.536" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.536" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.536" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.536" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.551" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:24.551" level="INFO">Unexpected frame: FC_PL_DATA</msg>
<msg timestamp="20181127 19:08:34.474" level="FAIL">10 seconds timeout</msg>
<msg timestamp="20181127 19:08:34.474" level="DEBUG">Traceback (most recent call last):
  File "C:\D\plc_system_test\lib\plc_tb_ctrl.py", line 325, in run_iot_test
    self.meter_platform_power_reset()
  File "C:\D\plc_system_test\lib\plc_tb_ctrl.py", line 1225, in meter_platform_power_reset
    self._deactivate_tb()
  File "C:\D\plc_system_test\lib\plc_tb_ctrl.py", line 493, in _deactivate_tb
    test_frame = self.tb_uart.wait_for_test_frame("DEACTIVATE_CNF_CMD")
  File "C:\D\plc_system_test\lib\plc_tb_uart.py", line 169, in wait_for_test_frame
    raise AssertionError(timeout_error)</msg>
<status status="FAIL" endtime="20181127 19:08:34.474" starttime="20181127 19:08:14.228"></status>
</kw>
<kw type="teardown" name="Record Test Status" library="plc_tb_ctrl">
<msg timestamp="20181127 19:08:34.474" level="INFO">begin 5 seconds waiting before exit...</msg>
<msg timestamp="20181127 19:08:40.475" level="INFO">stop detail logging</msg>
<status status="PASS" endtime="20181127 19:08:40.490" starttime="20181127 19:08:34.474"></status>
</kw>
<status status="FAIL" endtime="20181127 19:08:40.490" critical="yes" starttime="20181127 19:08:09.227">10 seconds timeout</status>
</test>
<status status="FAIL" endtime="20181127 19:08:40.490" starttime="20181127 19:08:09.136"></status>
</suite>
<status status="FAIL" endtime="20181127 19:08:40.490" starttime="20181127 19:08:09.120"></status>
</suite>
<statistics>
<total>
<stat fail="1" pass="0">Critical Tests</stat>
<stat fail="1" pass="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat fail="1" id="s1" name="Plc System Test" pass="0">Plc System Test</stat>
<stat fail="1" id="s1-s1" name="Tc.4" pass="0">Plc System Test.Tc.4</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
